### DISCUSSION TITLE
Ramys help me

### DISCUSSION THREAD

**Discussion by GeniShimada** [Ramys/Iptv-Brasil-2026#68] [Em Geral]
Deu certo o q vc passou 
Mas tou com um problema
Coloquei um software iptv num projetor q comprei 
Deu certo 
Só q 
Ele começa a transmitir 
Roda 5 s e volta ao inicio 
Ele é um projetor simples 
Mas tem uma tv box dentro ![image](https://github.com/Ramys/Iptv-Brasil-2023/assets/32342981/1d876d47-6478-4f8d-b500-d4060d1a9013)

![image](https://github.com/Ramys/Iptv-Brasil-2023/assets/32342981/6a468cca-47f7-4188-ab3c-06f5cf3447e9)

**Comment by GeniShimada** [comment]
Acho q deve ser problema de memória 
Mas ele carregou todo os menus de filmes séries e canais

**Reply by Ramys** [reply]
O ideal seria mesmo a tv box .

**Reply by GeniShimada** [reply]
Ramys 
Bom dia 
O projetor e também tv box 
Tá instalado na tv box 
Como faço para diminuir os canais 
Para testar se aliviando a memória funciona 
Tipo tirar os de Portugal

**Comment by GeniShimada** [comment]
![image](https://github.com/Ramys/Iptv-Brasil-2023/assets/32342981/d663259b-ea26-4daf-ae1d-cd0803d99a79)


![image](https://github.com/Ramys/Iptv-Brasil-2023/assets/32342981/cc1cb48f-e3b4-4ae4-9cd5-7cbf68887dbb)

![image](https://github.com/Ramys/Iptv-Brasil-2023/assets/32342981/6cdcf489-5a6d-4ad4-8914-ecf35861acf8)

**Comment by GeniShimada** [comment]
Aguardo 
Obrigado

**Comment by joaopauloriboli** [comment]
como faço para resolver um bug em relação aos canais ao vivo? a cada 10 segundos perde conexão mesmo com uma rede estável cabeada

**Reply by Ramys** [reply]
Teria que ter uma lista só sua com 1 ip.

**Reply by GeniShimada** [reply]
Como faço ?
Tirar aqueles canais portugueses tb 
Deve aliviar a memória

**Comment by operadorbrasil7233** [comment]
Olá queria falar com você em particular poderia me disponibilizar seu número tipo whatsapp?

**Reply by Ramys** [reply]
Bem pessoal..
Tem alguma dúvida?

**Reply by operadorbrasil7233** [reply]
Sim. Queria saber como você faz para cadastrar o login. Gostaria de tratar com você para lhe pagar algo mensalmente e você disponibilizar a mim o cadastramento de login e manter a lista atualizada. Sou provedor vpn e queria disponibilizar o reprodutor web dentro do meu app de forma gratuita aos clientes. Mais os que quiserem a url eu cobrarem um adicional de 2 reais. Mais queria ver um valor para você manter a lista atualizada e o cadastramento de login. Acha que seria possível?

**Comment by WarlenAA** [comment]
Salve Ramys, tudo na paz? Excelente conteúdo, obrigado por compartilhar conhecimento. Sou estudando de Análise e Desenvolvimento de Sistema, você conseguiria me enviar o código fonte desse seu projeto IPTV para meus estudos. Se sim, pode me enviar no email: warlenadriano39@gmail.com

muito obrigado e que Deus abençoe.

**Reply by Ramys** [reply]
Acho na internet e pesco..
Não trabalho com iptv.
